applications/crossbar/doc/ref/tasks.md

```diff
@@ -4,9 +4,6 @@
 
 #### Schema
 
-Key | Description | Type | Default | Required
---- | ----------- | ---- | ------- | --------
-`records` | List the rows of input data | `array(object)` |   | `true`
```

Comment:
If you have a custom schema that doesn't follow the convention, it should go direct into the `doc/tasks.md` version and not the auto-generated version.